ansible/tasks/postgres-extensions/09-pg-safeupdate.yml

```diff
@@ -28,3 +28,10 @@
   file:
     state: absent
     path: /tmp/pg-safeupdate-{{ pg_safeupdate_release }}
+
+- name: pg-safeupdate - set safeupdate.enabled
+  become: yes
+  lineinfile:
+    path: /etc/postgresql/postgresql.conf
+    state: present
+    line: safeupdate.enabled = off
```

Comment:
Why here instead of including in the base template?

Reply:
We do this for other extensions (`cron.database_name`, `auto_explain.log_min_duration`, etc.) so just did the same here

Reply:
I don't see a problem with including it in the base template though... going to smoke test that

Reply:
Seems to work fine - updated the PR